### PR TITLE
[WebDriver] Add "webSocketUrl" capability

### PR DESCRIPTION
**Pull request by whimboo**
This PR adds documentation for the `webSocketUrl` capability that is provided by the [WebDriver BiDi specification](https://w3c.github.io/webdriver-bidi/#establishing).

### CODE REVIEW

**Comment by whimboo** [comment]
@jgraham mind checking if the content for `webSocketUrl` looks fine to you? Thanks.

**Comment by whimboo** [comment]
Please note that I will take care of the missing pages for `WebDriver/Commands/NewSession` and `WebDriver/Commands/NavigateTo` in the next days.

File: files/en-us/web/webdriver/capabilities/websocketurl/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: webSocketUrl
+slug: Web/WebDriver/Capabilities/webSocketUrl
+tags:
+  - Reference
+  - WebDriver
+  - capabilities
+  - webSocketUrl
+---
+<p>With the <code>webSocketUrl</code> capability set to <code>true</code> a WebSocket server will be started in the browser, supporting bidirectional communication by using the <a href="https://w3c.github.io/webdriver-bidi/">WebDriver BiDi protocol</a>. This capability defaults to <code>false</code>, and has effect for the lifetime of the session.</p>
```
Comment (jgraham) [inline files/en-us/web/webdriver/capabilities/websocketurl/index.html:10]: I would remove the last sentence; "when set to true" implies that it must be set and the value must be true, and capabilities are always for the lifetime of the session.

File: files/en-us/web/webdriver/capabilities/websocketurl/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+When the capability is set to <code>true</code> the returned capabilities from the <a href="/en-US/docs/Web/WebDriver/Commands/NewSession">New Session</a> command will then have the same capability but with its value replaced by the WebSocket URL of the current WebDriver session.
```
Comment (jgraham) [inline files/en-us/web/webdriver/capabilities/websocketurl/index.html:12]: I might just say

"When the new session request has the `webSocketUrl` capability set to `true`, and the session starts successfully, the value of the `capabilities` field in the response will have a `webSocketUrl` property set to the URL of the WebSocket server."

**Review by jgraham — COMMENTED** [review]
This generally looks accurate and reasonable to me.

File: files/en-us/web/webdriver/capabilities/websocketurl/index.html (hunk 1 above)
Comment (whimboo) [reply files/en-us/web/webdriver/capabilities/websocketurl/index.html:10]: Sure, will remove it.

File: files/en-us/web/webdriver/capabilities/websocketurl/index.html (hunk 2 above)
Comment (whimboo) [reply files/en-us/web/webdriver/capabilities/websocketurl/index.html:12]: Makes sense. Thanks!

**Comment by Wikigoagec** [comment]
> This PR adds documentation for the `webSocketUrl` capability that is provided by the [WebDriver BiDi specification](https://w3c.github.io/webdriver-bidi/#establishing).

Version imagia'n6 beast network'

**Comment by Wikigoagec** [comment]
>

**Review by jgraham — APPROVED** [review]

**Comment by whimboo** [comment]
@teoli2003 could you please review?

**Comment by whimboo** [comment]
I talked with @sideshowbarker on Element and he will take care of the final review and merge once the branches are open again. Thanks!

File: files/en-us/web/webdriver/capabilities/websocketurl/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+slug: Web/WebDriver/Capabilities/webSocketUrl
+tags:
+  - Reference
+  - WebDriver
+  - capabilities
+  - webSocketUrl
+---
+<p>With the <code>webSocketUrl</code> capability set to <code>true</code> a WebSocket server will be started in the browser, supporting bidirectional communication by using the <a href="https://w3c.github.io/webdriver-bidi/">WebDriver BiDi protocol</a>.
+
+When the <a href="/en-US/docs/Web/WebDriver/Commands/NewSession">New Session</a> request has the <code>webSocketUrl</code> capability set to <code>true</code>, and the session starts successfully, the value of the capabilities field in the response will have a <code>webSocketUrl</code> property set to the URL of the WebSocket server.
```
Comment (sideshowbarker) [inline files/en-us/web/webdriver/capabilities/websocketurl/index.html:12]:
```suggestion
When the <a href="/en-US/docs/Web/WebDriver/Commands/NewSession">New Session</a> request has the <code>webSocketUrl</code> capability set to <code>true</code>, and the session starts successfully, the value of the <code>capabilities</code> field in the response will have a <code>webSocketUrl</code> property set to the URL of the WebSocket server.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**